package.json

```
@@ -25,7 +25,8 @@
     "test": "jest --verbose",
     "test:ui": "babel-node test/main.js",
     "bundlesize": "bundlesize",
-    "lokalise:download": "lokalise2 --token $LOKALISE_TOKEN --project-id $LOKALISE_PROJECT_ID file download --format json --unzip-to ./src/locales/ && node ./scripts/unwrap_lokalise.js"
+    "lokalise:download": "lokalise2 --token $LOKALISE_TOKEN --project-id $LOKALISE_PROJECT_ID file download --format json --unzip-to ./src/locales/ && node ./scripts/unwrap_lokalise.js",
+    "license-checker": "node_modules/.bin/license-checker-rseidelsohn --json > licenses.json --relativeLicensePath --relativeModulePath && prettier --write ./licenses.json"
```

You can safely remove `node_modules/.bin` here because `npm` will automatically look-up for binaries in that folder
```suggestion
    "license-checker": "license-checker-rseidelsohn --json > licenses.json --relativeLicensePath --relativeModulePath && prettier --write ./licenses.json"
```